.github/workflows/dataviz-autopublish.yml

```diff
@@ -0,0 +1,60 @@
+name: Dataviz module auto publish
```

Comment:
Do we want a autopublish for this package or should it be part of the TUI lifecycle ?